src/JNI/include/com/amazonaws/kinesis/video/producer/jni/Parameters.h

```diff
@@ -16,6 +16,8 @@ BOOL setStreamingEndpoint(JNIEnv* env, jstring streamingEndpoint, PCHAR pEndpoin
 BOOL setStreamDataBuffer(JNIEnv* env, jobject dataBuffer, UINT32 offset, PBYTE* ppBuffer);
 BOOL releaseStreamDataBuffer(JNIEnv* env, jobject dataBuffer, UINT32 offset, PBYTE pBuffer);
 BOOL setTags(JNIEnv *env, jobjectArray tagArray, PTag* ppTags, PUINT32 pTagCount);
+BOOL setStreamEventMetadata(JNIEnv* env, jobject streamEventMetadata, PStreamEventMetadata pStreamEventMetadata);
+BOOL allocStreamEventMetadataArray(JNIEnv* env, jobject streamEventMetadata, PCHAR* metaDataArray, jmethodID methodId);
```

Comment:
Do we need to declare `allocStreamEventMetadataArray` here if it is a helper function for `setStreamEventMetadata` and not used anywhere else? Can we just define it directly in Parameters.cpp?

Reply:
Yes, but then the order of the functions are placed in the file will matter, I'm ok with either way.